python/ray/serve/_private/replica.py

```diff
@@ -371,17 +372,36 @@ async def reconfigure(
         deployment_config: DeploymentConfig,
     ) -> Tuple[DeploymentConfig, DeploymentVersion]:
         try:
-            await self._user_callable_wrapper.reconfigure(deployment_config)
-            return await self._get_metadata()
+            user_config_changed = (
```

Comment:
Since `UserCallableWrapper` depends on autoscaling config, should `reconfigure()` also update autoscaling config?

Reply:
in theory it should; let me handle this in a separate PR where I am separating out the autoscaling metrics handling: https://github.com/ray-project/ray/pull/42043